Release/include/cpprest/details/http_client_impl.h

```diff
@@ -57,6 +57,11 @@ static utility::string_t flatten_http_headers(const http_headers &headers)
     utility::string_t flattened_headers;
     for(auto iter = headers.begin(); iter != headers.end(); ++iter)
     {
+        utility::string_t header_name = iter->first;
+        http_headers::_case_insensitive_cmp cmp;
```

Comment:
Alternatively, you could use utility::details::str_icmp() API for the comparison. Seems more straightforward than using the http_headers::_case_insensitive_cmp.